src/js/components/FilterButtons.js

```diff
@@ -62,19 +62,21 @@ class FilterButtons extends React.Component {
 }
 
 FilterButtons.defaultProps = {
-  renderButtonContent: function (title) {return title; },
-  onFilterChange: function () {}
+  inverseStyle: false,
+  onFilterChange: function () {},
+  renderButtonContent: function (title) {return title; }
```

Comment:
My linting is working like charm - eslint just don't complains about the missing space. :(